mountpoint-s3-client/src/s3_crt_client/get_object.rs

```diff
@@ -98,7 +99,10 @@ impl S3CrtClient {
                     }
                 },
                 move |offset, data| {
-                    _ = part_sender.unbounded_send(S3GetObjectEvent::BodyPart(offset, data.into()));
+                    _ = part_sender.unbounded_send(S3GetObjectEvent::BodyPart(GetBodyPart {
+                        offset,
+                        data: Bytes::copy_from_slice(data),
```

Comment:
Do we need to copy here?

Reply:
Because `data`'s lifetime is restricted to this callback. We are considering ways in which we could take ownership of the underlying buffer in the future and at that point we will be able to leverage [`Bytes::from_owner`](https://docs.rs/bytes/latest/bytes/struct.Bytes.html#method.from_owner) to avoid the copy.